go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/Skyscanner/argocd-progressive-rollout
 go 1.15
 
 require (
-	github.com/argoproj/argo-cd v0.8.1-0.20210218202601-6de3cf44a4cb
+	github.com/argoproj/argo-cd v0.8.1-0.20210218100039-a4ee25b59d8d
```

Comment:
If you are only adding some scripts, why is go.mod changing?

Reply:
I think it moved the needle when i ran go-mod-hack, I can omit it from the pr if you want, but it will probably be changing all the time.